File: exec2.py
import os
import sys

if len(sys.argv) != 2:
    print(f"Usage : {sys.argv[0]} chemin_de_destination")
    exit(1)

pwd = os.getenv("PWD")

os.execl("/usr/bin/cp", "cp", "-rf", pwd, sys.argv[1])

File: exec2.c
#include<stdio.h>
#include<stdlib.h>
#include<unistd.h>

int main(int argc, char** argv)
{
    char* pwd = getenv("PWD");

    if (argc != 2)
    {
        printf("Usage : %s <chemin de destination>\n", argv[0]);
        return -1;
    }

    execl("/usr/bin/cp", "cp", "-rf", pwd, argv[1], NULL);

    return 0;
}